shuffle/cli/src/account.rs

```diff
@@ -174,6 +201,58 @@ pub fn create_account_onchain(
     Ok(())
 }
 
+async fn handle_account_creation_on_network(
+    network: &Network,
+    new_account: &LocalAccount,
+    test_account: &LocalAccount,
+) -> Result<()> {
```

Comment:
Similar to above, I'm not a fan of creating a method simply to invoke another method twice. This method would be much more reusable if it handled on account at a time. I recommend having this simply take `account: &LocalAccount` and invoking it twice, once for new_account, and once for test_account.

We'll be doing another pass on this when we have more/alternate # of accounts to create (Alice, Bob, Charlie).

Reply:
Changing this to handle only one account!